<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xiugaixingxi</class>
 <widget class="QWidget" name="xiugaixingxi">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>318</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>389</width>
    <height>318</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>389</width>
    <height>318</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>xiugaixingxi</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="touxiang">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Xiugai_Displaybt">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>修  改 昵 称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>修改个性签名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nichengxiugai">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="gexingqianming">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="queding">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nichengqingchubt">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gexingqianmingbt">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="guanbi">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="man_radioButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>145</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>性        别：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="woman_radioButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="denglu.qrc"/>
 </resources>
 <connections/>
</ui>
